signac/contrib/project.py

```diff
@@ -1860,38 +1843,16 @@ class JobsCursor:
     project : :class:`~signac.Project`
         Project handle.
     filter : Mapping
-        A mapping of key-value pairs that all indexed job state points are
-        compared against (Default value = None).
-
-    Notes
```

Comment:
Ah this is what I was remembering. Where is `{}` being treated differently from `None`? In the code above, we just check `if not filter`, which will behave identically for `None` and `{}`. It looks like on line 1857 below we added logic to replace `{}` with `None`, should the comment here have been removed when that was added (I don't remember it being there)? It looks like this issue is specific to when the parameter is passed to building a `JobsCursor`, which is why it's not relevant above, is that correct?

Reply:
This was changed in #514 and there is no longer a performance difference between `{}` and `None`. Yes, you are correct that the comment about performance should have been removed at that time.